README.md

```diff
@@ -98,3 +98,31 @@ hvac_zones = orm.session.query(Location)\
                         .all()
 print(f"Model has {len(hvac_zones)} HVAC Zones")
 ```
+## Validate with Shape Constraint Language
+
+The module utilizes the [pySHACL](https://github.com/RDFLib/pySHACL) package to validate a building ontology
+against the Brick Schema, its default constraints (shapes) and user provided shapes.
+
+```python
+from brickschema.validate import Validate, ResultsSerialize
+from rdflib import Graph
+import sys
+
+dataG = Graph()
+dataG.parse('myBuilding.ttl', format='turtle')
+v = Validate()
+v.addShapeFile('extraShapes.ttl')
+(conforms, results_graph, results_text) = v.validate(dataG)
+sys.stdout.write(results_text)
```

Comment:
We should be using `print` rather than `sys.stdout.write` here, and we should probably only print if the instance does not conform

Reply:
Print is fine.  But results_text actually contains something like "conforms" when all is well.  And the pyshacl command does print it out.  So we probably should, too.